src/main/java/page_objects/LoginPage.java

```diff
@@ -25,6 +25,15 @@ public void login(Account account) {
         this.btnLogin.click();
     }
 
+    public void loginNTimes(Account account, int n) {
+        for(int i=0;i<n;i++){
+            this.txtEmail.enterText(account.getEmail());
```

Comment:
DRY.

Reply:
Sorry, i will call function login to replace code